### ISSUE TITLE
Package manager support

### ISSUE THREAD

**Issue by schwa** [freedmand/textra#2]
Seems like a great project that would benefit from submission to home-brew?

**Comment by freedmand** [comment]
I agree! I think once the API's more stable I will focus more on that.

**Comment by trevormeier** [comment]
Agreed! I'd love to see this included in home-brew

**Comment by chicks-net** [comment]
Are the API's more stable at this point?
